orderbook/src/database/orders.rs

```diff
@@ -78,6 +78,19 @@ impl Database {
             .map(|_| ())
     }
 
+    pub async fn cancel_order(&self, order: &Order) -> Result<()> {
+        const QUERY: &str = "\
+            UPDATE orders
+            SET invalidated = true \
```

Comment:
To reduce future db updates maybe we can still persist the timestamp but only return a boolean to the caller of the API.

Also I don't think orders has an invalidated field yet, so you would need to create a db migration for it anyways.